src/ansi-c/Makefile

```diff
@@ -48,15 +49,24 @@ CLEANFILES = ansi-c$(LIBEXT) \
   ansi_c_y.output \
   library/converter$(EXEEXT) cprover_library.inc \
   file_converter$(EXEEXT) library_check \
-  gcc_builtin_headers_generic.inc gcc_builtin_headers_ia32.inc \
-  arm_builtin_headers.inc cw_builtin_headers.inc \
-  gcc_builtin_headers_arm.inc gcc_builtin_headers_alpha.inc \
-  gcc_builtin_headers_mips.inc gcc_builtin_headers_power.inc \
-	clang_builtin_headers.inc gcc_builtin_headers_ia32-2.inc \
-  gcc_builtin_headers_math.inc gcc_builtin_headers_omp.inc \
-  gcc_builtin_headers_tm.inc gcc_builtin_headers_ubsan.inc \
+  arm_builtin_headers.inc \
+  clang_builtin_headers.inc \
+  cw_builtin_headers.inc \
+  gcc_builtin_headers_alpha.inc \
+  gcc_builtin_headers_arm.inc \
+  gcc_builtin_headers_generic.inc \
+  gcc_builtin_headers_ia32-2.inc \
+  gcc_builtin_headers_ia32-3.inc \
+  gcc_builtin_headers_ia32-4.inc \
+  gcc_builtin_headers_ia32.inc \
+  gcc_builtin_headers_math.inc \
   gcc_builtin_headers_mem_string.inc \
-  gcc_builtin_headers_ia32-3.inc gcc_builtin_headers_ia32-4.inc
+  gcc_builtin_headers_mips.inc \
+  gcc_builtin_headers_omp.inc \
+  gcc_builtin_headers_power.inc \
+  gcc_builtin_headers_tm.inc \
+  gcc_builtin_headers_types.inc \
+  gcc_builtin_headers_ubsan.inc
```

Comment:
While at this, would it be possible to introduce a Makefile variable holding all the header files? As is, the same list of files appears three times.

Reply:
done